### DISCUSSION TITLE
[求助]有没有推荐的爱情片？

### DISCUSSION THREAD

**Discussion by WORLD-executeME** [share121/inter-knot#418] [其他]
![屏幕截图 2024-08-09 211305](https://github.com/user-attachments/assets/cff2ddbc-620d-446b-9c91-7710d4ff5108)

不是快到七夕了吗？想说找几盘爱情片的录像带看看，但本人不太常看爱情片，只跟着家人朋友看过一两部经典的，最近有点忙没时间自己去录像店慢慢问，想听听绳网的
大家有什么推荐的录像带，《最后一次飞行》如何？

**Comment by ShangHuaWenYuan** [comment]
推荐看《可爱伊埃斯爱上我》，甜度拉满，嘿嘿嘿我的伊埃斯🤤🤤🤤

**Reply by WORLD-executeME** [reply]
真的有这个片子吗？

**Comment by TTGFCCt** [comment]
这边推荐《噢～甜心》

**Reply by WORLD-executeME** [reply]
和朋友看过了，莫妮卡大姐姐很好看

**Comment by kano0x0** [comment]
没人比六分街录像店的店主更懂录像带

**Reply by WORLD-executeME** [reply]
也是，我还是抽空去问问吧

**Comment by Qiling-XwX** [comment]
这边建议看雷雨

**Reply by WORLD-executeME** [reply]
在学校的语文课上看过了

**Comment by sandgook** [comment]
水评论+1

**Reply by WORLD-executeME** [reply]
...

**Comment by qiu-1214** [comment]
六分街的「Random Play」录像店进的带子水准很高，你可以去找找
